---
layout: page
title: "Léo Hardy and Olga Soutourina"
permalink: /speakers/January/
---

<span style="font-size: 1.5em;"><strong>Léo Hardy</strong></span><br>
Zeynep Baharoglu's lab, Epitranscriptomic and translational responses to antibacterial stress Team, Expression Génétique Microbienne, CNRS UMR8261, Institut Pasteur, Université Paris Cité, Institut de Biologie Physico-Chimique

**Title**:  
Investigating the role of the ribosomal large subunit pseudouridine synthase B (RluB) in tobramycin tolerance in Vibrio cholerae.

**Abstract**:  
The massive and inappropriate use of antibiotics has led to their widespread dissemination in the environment, where they are encountered as concentration gradients. While many studies have addressed the effects of lethal antibiotic doses on bacterial physiology, far less is known about how bacteria respond to so-called subinhibitory concentrations (i.e., levels below the minimal inhibitory concentration), even though these are likely the levels most frequently experienced in natural settings. Using Vibrio cholerae, a bacterium that alternates between aquatic reservoirs and the human host and therefore faces strong environmental and antibiotic pressures, we found that rRNA modification enzymes play a major role in adaptation to subinhibitory antibiotic exposure.
In this study, we focused on one enzyme in particular, RluB, a pseudouridine synthase catalysing the formation of pseudouridine at position 2588 of the 23S rRNA in V. cholerae. We showed that deletion of rluB leads to increased tolerance to tobramycin, an antibiotic targeting the 30S ribosomal subunit. Analysis of catalytic mutants of RluB further indicated that this phenotype is not driven by the absence of the rRNA modification itself, but rather by the presence of RluB as a protein. Preliminary experiments suggest that ribosomes from the ΔrluB mutant may display increased translational fidelity but reduced stability. Ongoing work combines epistasis analyses with other ribosomal components, reporter fusions to quantify translation efficiency, sucrose gradient analyses of ribosome integrity, and omics approaches to define the molecular basis of this phenotype and to clarify how RluB contributes to bacterial adaptation to subinhibitory aminoglycoside exposure.


<span style="font-size: 1.5em;"><strong>Olga Soutourina</strong></span><br>

**Title**:  
RNomics during C. difficile infection cycle: from identification to function
Team "Regulatory RNAs in Clostridia" , Université Paris-Saclay, Institute for Integrative Biology of the Cell (I2BC), UMR 9198 

**Abstract**:  
Clostridioides difficile is the leading cause of healthcare associated diarrhoea in industrialized countries. Many aspects of C. difficile pathogenesis associated with adaptation strategies remain poorly understood. Non-coding RNAs (ncRNAs) shape virulence in many pathogens and modulate host responses, however, their role in C. difficile infection has not been explored. Clostridia have developed unique RNA-based mechanisms and associated machineries (e.g. RNA chaperone protein Hfq), which differ from both Gram-positive Bacillota like bacilli and staphylococci, and well-studied Gram-negative enterobacteria. Studying this ancient group of bacteria should reveal other strategies of adaptation and insights into the evolutionary history of bacterial riboregulation. Using bioinformatics and genome-wide approaches we identified more than 200 ncRNAs in C. difficile, the majority being specific to this pathogen. Our recent dual RNA-seq data show that many of ncRNAs are expressed during infection, indicating that they contribute to shape virulence and adaptation of C. difficile to its host. Some of these ncRNAs participate in the regulation of sporulation process in accordance with accumulating evidence for the importance of RNA-based mechanisms in the control of this key step in C. difficile infection cycle including Hfq and Hfq-binding ncRNAs. To control genetic exchanges during the infection cycle, C. difficile survives in phage-rich gut communities by relying on defense systems like CRISPR-Cas, Toxin-Antitoxin (TA) and abortive infection (Abi) contributing to prophage maintenance, prevention of phage infection, and stress response. RNAs have emerged as key components of these systems including CRISPR RNAs and antitoxin RNAs within type I and type III TA. Overall, these data are shedding new light on the riboregulation mechanisms in C. difficile opening new avenues for specific therapeutic and diagnostic strategies.
